src/main/java/com/thebois/models/world/World.java

```diff
@@ -120,15 +112,7 @@ public Object find() {
      * @return The list to be returned.
      */
     public Collection<IStructure> getStructures() {
-        final Collection<IStructure> copy = new ArrayList<>();
-        for (final IStructure[] matrix : structureMatrix) {
-            for (final IStructure structure : matrix) {
-                if (structure != null) {
-                    copy.add(structure.deepClone());
-                }
-            }
-        }
-        return copy;
+        return MatrixUtils.toCollection(this.structureMatrix);
```

Comment:
Does this make them copies?

Reply:
Don't think so, but they shouldn't be copies, right? Since a `Structure` is an entity with its own state, it's not a value object.